docs/tutorials/stateful-application/zookeeper.md

```diff
@@ -462,17 +445,16 @@ volumeClaimTemplates:
 ```
 
 
-The StatefulSet controller generates a PersistentVolumeClaim for each Pod in 
-the StatefulSet. 
+The `StatefulSet` controller generates a `PersistentVolumeClaim` for each Pod in
+the `StatefulSet`.
 
-Get the StatefulSet's PersistentVolumeClaims.
+Use the following command to get the `StatefulSet`'s `PersistentVolumeClaims`.
 
 ```shell
 kubectl get pvc -l app=zk
 ```
 
-When the StatefulSet recreated its Pods, the Pods' PersistentVolumes were 
-remounted.
+When the `StatefulSet` recreated its Pods, it remounts the Pods' `PersistentVolumes`.
```

Comment:
I'd get rid of the code markup for PersistentVolumes here, and maybe even make it two words (Persistent Volumes). Calling out K8s objects is tricky; another general solution is "`PersistentVolume` objects" -- but saying that you remount an object sounds wrong too.